addons/mail/static/tests/chat_window/chat_window.test.js

```diff
@@ -187,14 +187,14 @@ test("chat window: basic rendering", async () => {
     await contains("[title='Open Actions Menu']");
     await click("[title='Open Actions Menu']");
     await contains(".o-mail-ChatWindow-command", { count: 15 });
+    await contains(".o-dropdown-item", { text: "Open in Discuss" });
```

Comment:
Note: the order of the contains do not determine the order in the DOM, in other words, moved or not, this test just checks presence not sequence.

Reply:
Yes, I know it, I took the opportunity to change because it felt better, though yes the ordering is not really checked in the test.